adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -363,10 +363,6 @@ - (nullable NSError *)renderBannerAdInView:(nonnull UIView *)bannerView
       GADMAdapterVungleMutableDictionarySetObjectForKey(options, VunglePlayAdOptionKeyUser,
                                                         extras.userId);
     }
-    if (extras.ordinal) {
```

Comment:
Just to confirm, is this parameter not supported anymore?

If that's the case, we would need to update our developer documentation to reflect that: https://developers.google.com/admob/ios/mediation/vungle#network-specific_parameters.

Reply:
Yes this is no-longer supported as of 6.12. @joshh-devrel